### PR TITLE
New brig map

### PR DESCRIPTION
**Pull request by utahclock**
creates a new brig layout

### CODE REVIEW

**Comment by utahclock** [comment]
[img src="http://puu.sh/o3xcZ/5a774587ae.png"]

**Comment by EMT321** [comment]
Is there an isolation perma?

**Comment by utahclock** [comment]
yeah

**Comment by EMT321** [comment]
Could we possibly get some more images, showing the whole thing and the space outside?

File: html/changelogs/utahclock-newbrig [hunk 1]
```diff
@@ -33,5 +33,5 @@ delete-after: True
 # Also, this gets changed to [] after reading.  Just remove the brackets when you add new shit.
 # Please surround your changes in  double quotes ("). It works without them, but if you use certain characters it screws up compiling. The quotes will not show up in the changelog.
 changes: 
-  - rscadd: "Added a ghost sprite for donators"
+  - rscadd: "adds new brig layout"
```
Comment (Cruix) [inline html/changelogs/utahclock-newbrig:37]: What
Why did this rename a file instead of just adding a new one
What

**Pull request closed without merging**